backend/lib/edgehog/devices/device/manual_actions/send_create_container.ex

```diff
@@ -35,7 +35,7 @@ defmodule Edgehog.Devices.Device.ManualActions.SendCreateContainer do
     device = changeset.data
 
     with {:ok, container} <- Ash.Changeset.fetch_argument(changeset, :container),
-         {:ok, container} <- Ash.load(container, [:env_encoding, :image]),
+         {:ok, container} <- Ash.load(container, [:env_encoding, :image, networks: [:id]]),
```

Comment:
very nit:
```suggestion
         {:ok, container} <- Ash.load(container, [:env_encoding, :image, :networks]),
```
calling `Ash.load` with just `:networks` works, as networks ids are not a nested calculation or relationship.